### ISSUE TITLE
[Bug]: When selecting "Reb. vs Occ." as game mode, the invaders will still appear after restarting the server

### ISSUE THREAD

**Issue by Drift91** [igorkis-scrts/A3-Antistasi-Plus#93]
### Describe the bug

If you select "Reb. vs Occ." as the game mode, after restarting the server or entering the command `#restart` as admin, the invaders will appear on the map as if "Reb. vs Occ. vs Inv."

### How to reproduce

Start a new save with the game mode set to "Reb. vs Occ."
Make a persistent save.
Restart the mission with `#restart` or fully restart the server.
Load up the same save.
Check the map. On Altis they'll be in the North-East peninsula, as they normally would be if they were enabled.

### Version

2.0.2

### Have you altered the code?

No

### What i have changed

```markdown
Although I have altered the code on my live server, I tested with the official Steam release as well with a new save and still experienced the issue.
```

### Map

Altis

### What server?

Private dedicated server

### Mods

```html
"-mod=@Antistasi_Plus_2_0;@CBA_A3;@ACE__No_medical_Updated_;@RHSAFRF;@RHSUSAF;@RHSGREF;@RHSSAF;@ACE_Compat__RHS_AFRF;@ACE_Compat__RHS_USAF;@ACE_Compat__RHS_GREF;@ACE_Compat__RHS_SAF;@3CB_Factions;@Task_Force_Arrowhead_Radio_BETA__;"
```

**Comment by igorkis-scrts** [comment]
Known, will be fixed in the next patch.

**Comment by Drift91** [comment]
Alright, glad to hear that.

**Comment by igorkis-scrts** [comment]
Fixed in 2.1.
